From c45c07fc722a25df52017a3771936e7494d1cb14 Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Wed, 19 Nov 2014 16:12:49 -0500
Subject: [ethernet] be2net: fix some log messages

Message-id: <1416413587-12091-2-git-send-email-ivecera@redhat.com>
Patchwork-id: 100328
O-Subject: [RHEL7.1 PATCH 01/19] be2net: fix some log messages
Bugzilla: 1165755
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

BZ: #1165755

Upstream commit(s):
commit 50762667f78e346c343544f88270c0a831789e7d
Author: Vasundhara Volam <vasundhara.volam@emulex.com>
Date:   Fri Sep 12 17:39:14 2014 +0530

    be2net: fix some log messages

    This patch fixes the following minor issues with log messages in be2net:
      1) Period is not required at the end of log message.
      2) Remove "Unknown grp5 event" logs to reduce noise. The driver can safely
         ignore async events from FW it's not interested in.
      3) Reword a log message for better readability to say that SRIOV
         "is disabled" rather than "not supported".

    Signed-off-by: Vasundhara Volam <vasundhara.volam@emulex.com>
    Signed-off-by: Sathya Perla <sathya.perla@emulex.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/emulex/benet/be_cmds.c b/drivers/net/ethernet/emulex/benet/be_cmds.c
index 7d496d4..1fcd69e 100644
--- a/drivers/net/ethernet/emulex/benet/be_cmds.c
+++ b/drivers/net/ethernet/emulex/benet/be_cmds.c
@@ -309,8 +309,6 @@ static void be_async_grp5_evt_process(struct be_adapter *adapter,
   be_async_grp5_pvid_state_process(adapter, compl);
   break;
  default:
-  dev_warn(&adapter->pdev->dev, "Unknown grp5 event 0x%x!\n",
-    event_type);
   break;
  }
 }
diff --git a/drivers/net/ethernet/emulex/benet/be_ethtool.c b/drivers/net/ethernet/emulex/benet/be_ethtool.c
index 32b432f..ef31b7d 100644
--- a/drivers/net/ethernet/emulex/benet/be_ethtool.c
+++ b/drivers/net/ethernet/emulex/benet/be_ethtool.c
@@ -676,7 +676,7 @@ be_set_pauseparam(struct net_device *netdev, struct ethtool_pauseparam *ecmd)
  status = be_cmd_set_flow_control(adapter,
       adapter->tx_fc, adapter->rx_fc);
  if (status)
-  dev_warn(&adapter->pdev->dev, "Pause param set failed.\n");
+  dev_warn(&adapter->pdev->dev, "Pause param set failed\n");
 
  return be_cmd_status(status);
 }
diff --git a/drivers/net/ethernet/emulex/benet/be_main.c b/drivers/net/ethernet/emulex/benet/be_main.c
index 21fa29a..fbe179d 100644
--- a/drivers/net/ethernet/emulex/benet/be_main.c
+++ b/drivers/net/ethernet/emulex/benet/be_main.c
@@ -1093,6 +1093,7 @@ static int be_change_mtu(struct net_device *netdev, int new_mtu)
  */
 static int be_vid_config(struct be_adapter *adapter)
 {
+ struct device *dev = &adapter->pdev->dev;
  u16 vids[BE_NUM_VLANS_SUPPORTED];
  u16 num = 0, i = 0;
  int status = 0;
@@ -1114,16 +1115,15 @@ static int be_vid_config(struct be_adapter *adapter)
   if (addl_status(status) ==
     MCC_ADDL_STATUS_INSUFFICIENT_RESOURCES)
    goto set_vlan_promisc;
-  dev_err(&adapter->pdev->dev,
-   "Setting HW VLAN filtering failed.\n");
+  dev_err(dev, "Setting HW VLAN filtering failed\n");
  } else {
   if (adapter->flags & BE_FLAGS_VLAN_PROMISC) {
    /* hw VLAN filtering re-enabled. */
    status = be_cmd_rx_filter(adapter,
         BE_FLAGS_VLAN_PROMISC, OFF);
    if (!status) {
-    dev_info(&adapter->pdev->dev,
-      "Disabling VLAN Promiscuous mode.\n");
+    dev_info(dev,
+      "Disabling VLAN Promiscuous mode\n");
     adapter->flags &= ~BE_FLAGS_VLAN_PROMISC;
    }
   }
@@ -1137,11 +1137,10 @@ set_vlan_promisc:
 
  status = be_cmd_rx_filter(adapter, BE_FLAGS_VLAN_PROMISC, ON);
  if (!status) {
-  dev_info(&adapter->pdev->dev, "Enable VLAN Promiscuous mode\n");
+  dev_info(dev, "Enable VLAN Promiscuous mode\n");
   adapter->flags |= BE_FLAGS_VLAN_PROMISC;
  } else
-  dev_err(&adapter->pdev->dev,
-   "Failed to enable VLAN Promiscuous mode.\n");
+  dev_err(dev, "Failed to enable VLAN Promiscuous mode\n");
  return status;
 }
 
@@ -3383,7 +3382,7 @@ static int be_get_sriov_config(struct be_adapter *adapter)
 
  if (!be_max_vfs(adapter)) {
   if (num_vfs)
-   dev_warn(dev, "device doesn't support SRIOV\n");
+   dev_warn(dev, "SRIOV is disabled. Ignoring num_vfs\n");
   adapter->num_vfs = 0;
   return 0;
  }
@@ -3657,7 +3656,7 @@ static int be_setup(struct be_adapter *adapter)
  dev_info(dev, "FW version is %s\n", adapter->fw_ver);
 
  if (BE2_chip(adapter) && fw_major_num(adapter->fw_ver) < 4) {
-  dev_err(dev, "Firmware on card is old(%s), IRQs may not work.",
+  dev_err(dev, "Firmware on card is old(%s), IRQs may not work",
    adapter->fw_ver);
   dev_err(dev, "Please upgrade firmware to version >= 4.0\n");
  }
-- 
1.7.1